app/src/main/java/com/example/cats_or_dogs2/CatAdapter.java

```diff
@@ -43,15 +43,15 @@ public viewHolder(@NonNull View itemView) {
     @NonNull
     @Override
     public CatAdapter.viewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
-        View view = LayoutInflater.from(context).inflate(R.layout.activity_table_view,parent,false);
+        View view = LayoutInflater.from(context).inflate(R.layout.list_row_item,parent,false);
```

Comment:
The adapter will be responsible for inflating a series of `list_row_item`s within the `RecyclerView`, so we need to inflate that view here rather than the enclosing activity's view.  That way the `findViewById()` method call will be able to find the views we're looking for.